modules/gke-cluster/variables.tf

```diff
@@ -41,31 +41,12 @@ variable "enable_kubernetes_alpha" {
   default = "false"
 }
 
-variable "oauth_scopes" {
-  type = "list"
-
-  default = [
-    "https://www.googleapis.com/auth/compute",
-    "https://www.googleapis.com/auth/devstorage.read_only",
-    "https://www.googleapis.com/auth/logging.write",
-    "https://www.googleapis.com/auth/monitoring",
-  ]
-}
-
-variable "node_type" {
-  default = "n1-standard-2"
-}
-
-variable "node_image_type" {
-  default = "COS"
-}
-
 variable "initial_node_count" {
-  default = 2
+  default = 1
 }
 
 variable "kubernetes_version" {
-  default = "1.10.5-gke.0"
+  default = "1.12.5-gke.10"
```

Comment:
I'm not sure I would do the k8s upgrade at the same time as this node_pool upgrade. Perhaps a (separate) k8s upgrade would be a good first test of the new node_pool strategy?

Reply:
Thanks, I'm with you here, but this does not affect us, it's just the default version update. We set the version in gpii-infra module -  https://github.com/gpii-ops/gpii-infra/blob/7369833e9bd01f3fba805d0b2f2bdc7f60812c7d/gcp/modules/gke-cluster/main.tf#L29

Reply:
Ah, of course.